ReactAndroid/src/main/java/com/facebook/react/modules/image/ImageLoaderModule.java

```diff
@@ -81,22 +82,23 @@ public void getSize(
     Uri uri = Uri.parse(uriString);
     ImageRequest request = ImageRequestBuilder.newBuilderWithSource(uri).build();
 
-    DataSource<CloseableReference<CloseableImage>> dataSource =
-      Fresco.getImagePipeline().fetchDecodedImage(request, mCallerContext);
+    DataSource<CloseableReference<PooledByteBuffer>> dataSource =
+      Fresco.getImagePipeline().fetchEncodedImage(request, mCallerContext);
```

Comment:
@orenklein @foghina 
See my approach at https://github.com/bood/react-native/commit/ea4b4dd8b6dbcf725ef34ae75c2ce36861229bfe

I'll make another PR if it looks good.